nservicebus/sagas/saga-finding.md

```diff
@@ -11,7 +11,7 @@ related:
 - nservicebus/sagas/concurrency
 ---
 
-A saga can handle multiple messages. When NServiceBus receives a message that should be handled by a saga, it uses the [configured mapping information](/nservicebus/sagas/#correlating-messages-to-a-saga) to determine the correct saga instance that should handle the incoming message. However, in some complex scenarios, one property in the message may not be sufficient to resolve a saga instance. For example, two or more properties might together identify the correct saga instance. In such cases it becomes necessary to define a custom saga finder.
+A saga can handle multiple messages. When NServiceBus receives a message that should be handled by a saga, it uses the [configured mapping information](/nservicebus/sagas/#correlating-messages-to-a-saga) to determine the correct saga instance that should handle the incoming message. In many cases the correlation logic is simple and can be specified using the `ConfigureHowToFindSaga` method, which is the recommended default approach. However, if the correlation logic is very complex it might be necessary to define a custom saga finder.
```

Comment:
When @SimonCropp and I modified this, we consciously chose not to use `ConfigureHowToFindSaga`, i.e. method names, as if it changes, it adds overhead. 

>In many cases the correlation logic is simple and can be specified using the `ConfigureHowToFindSaga` method, which is the recommended default approach. However, if the correlation logic is very complex it might be necessary to define a custom saga finder.

How about this:
Letting NServiceBus correlate the sagas based on the mapping information is the recommended approach.  However in scenarios where the correlation logic is complex, one property in the message may not be sufficient to resolve a saga instance. For example, two or more properties might together identify the correct saga instance. In such cases, it becomes necessary to define a custom saga finder.